doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,227 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
+
 =============================
 Building Chapel Documentation
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
-The easiest way to build the Chapel web documentation is to use the top level
-make target::
-
-   [g]make docs
+The easiest way to build the Chapel web documentation is to run the command
+``make docs`` in the top level directory of your ``chapel`` Github clone.
 
 This will build the documentation for the standard and internal modules using
 chpldoc, and symlink all of the ``$CHPL_HOME/doc/`` documents over to
 a sphinx project that lives in ``$CHPL_HOME/doc/``. Once built,
 the documentation can be viewed at ``$CHPL_HOME/doc/html/index.html``.
+
+The html documentation is also generated by the Github CI (continuous integration)
+process.  You can go to your pull request in Github, click on "Checks",
+then "Ci" (on the left), and then scroll down to see the "documentation"
+(at the very bottom).  It enables you to download a zip file of the built
+documentation.
+
+Recent and previous releases of the documentation can be found at
+https://chapel-lang.org/docs/.
+
+
+How the HTML is Generated
+=========================
+
+The Chapel documentation shown at https://chapel-lang.org/docs/ is generated
+in three different ways:
+
+#.  reStructuredText files (with a .rst extension) are converted into .html
+    files via sphinx (https://sublime-and-sphinx-guide.readthedocs.io/).
+#.  Comments and code from Chapel source files are converted into .rst files
+    using ``chapel/doc/util/chpl2rst.py``.
+#.  Some comments and code from Chapel source files are converted into .rst
+    files using ``chapel/tools/chpldoc/``, which will leave out TODOs,
+    FIXMEs, and other parts of the comments.
```

Comment:
In general, I think we only apply one of these last two techniques to any given `.chpl` file, so for bullet 3, you could say "Comments and code for Chapel module files are converted into..." for the third bullet and "...from Chapel examples, such as the primers, are converted into..." for the second (and I might swap the order because I think more developers are likely to run into the need to update and test module documentation than primer-based documentation, so it'd optimize for the common case).

Reply:
Swapped the order.